extensions/das/artifacts/metering-dashboard/capps/ppaas-metering-service/GadgetMemberCount/Member_Count/js/gadgetconf.js

```diff
@@ -16,7 +16,6 @@
 var gadgetConfig = {
     "id": "Member_Count",
     "title": "Member_Count",
-    "datasource": "MEMBER_COUNT",
```

Comment:
Why did we remove this variable? I understand that we needed to remove the table name from the request parameter. Is this something we added initially?